lib/adal-angular.js

```diff
@@ -320,7 +320,7 @@ if (typeof module !== 'undefined' && module.exports) {
                             if (authService.loginInProgress()) {
                                 authService.info('login already start.');
                                 return $q.reject();
-                            } else if (authService.config && isEndpoint) {
+                            } else if (authService.config && (!authService.config.endpoints || isEndpoint)) {
```

Comment:
@sarmut We already check at line 297 if the given uri matches any configured endpoints or app's client_id or not. We are again checking it in line 311-317 which is not needed. I think we can get of lines 311-317 and change the `else if` check to just `else` since we want to make that request for both external endpoints and app's backend. Also, please update the comment below that check.